### ISSUE TITLE
SCANsat ignores resources without a GLOBAL_RESOURCE block

### ISSUE THREAD

**Issue by Starstrider42** [KSPModStewards/SCANsat#218]
While experimenting with some planetary resource configs, I found a strange bug: if a resource does not have a `GLOBAL_RESOURCE` block, SCANsat will not display that resource on either the large map or the overlay. This happens even if the resource has `PLANETARY_RESOURCE` or `BIOME_RESOURCE` blocks, and the stock resource system confirms that the resource is present. (A good example if you don't want to mess around with configs is Karborundum in the default CRP config).

Please allow SCANsat to display resources that use any of the three planetary resource declarations.

**Comment by DMagic1** [comment]
Adding planetary resources should be easy, it checks them anyway. SCANsat doesn't touch biome resource configs, so I'm not sure how much trouble it would be to account for them.

Is there any reason to use the configs like this, rather than just giving them a global config with all values set to zero?

**Comment by Starstrider42** [comment]
Yes, as far as I can tell an empty global config works around this bug with no side effects -- but the config-maker has to know about the bug in the first place, since it's not a very natural approach if you only want a resource to appear in specific circumstances. Not requiring a global node would keep the issue from being re-reported with mods X, Y, and Z.

**Comment by Starstrider42** [comment]
As of SCANsat 20.4, this issue seems to have been fixed -- in fact, even resources that have only a biome-level config appear in the UI. Closing as fixed.
